website/src/components/product/CustomerLogosSection.tsx

```diff
@@ -26,10 +33,17 @@ const ITEMS: (
     {
         name: 'Convoy',
         url: '/external-logos/convoy-logo.svg',
+        link: {
+            url: '/case-studies/convoy-improved-on-boarding',
+        },
     },
     {
         name: 'Yelp',
         url: '/external-logos/yelp.svg',
+        link: {
+            url: 'https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html',
+            target: '_blank',
```

Comment:
could we also add `rel: 'nofollow'`   prevent SEO for external sites.

Reply:
Why `nofollow`? We link to that blog post in several other parts of the site as well.